app/models/site_setting.rb

```diff
@@ -8,6 +8,7 @@ class SiteSetting < ActiveRecord::Base
   validates_presence_of :data_type
 
   after_save do |site_setting|
+    SiteSetting.refresh!
```

Comment:
Why is the refresh needed?

Reply:
Without `refresh!` method if I call  like `SiteSetting.UPDATED_SETTING_NAME` then it's returning the old value. I think in my particular case I can use like this https://github.com/discourse/discourse/pull/6705/commits/1091a5b80c51acf45c81b1bd4afc534df523ee01